sample/android/build.gradle

```diff
@@ -2,13 +2,17 @@
 
 buildscript {
     ext {
+        buildToolsVersion = "30.0.2"
+
         minSdkVersion = 21
         compileSdkVersion = 31
         targetSdkVersion = 31
+
+        ndkVersion = "21.4.7075529"
```

Comment:
Why do we need these 2 changes? the tooling should be able to pick up the default versions

Reply:
These came over when I copied everything from the upgrade helper, I assume because RN updated their template. Do you say we remove them @marandaneto?